src/components/TooltipNew/design-guide.md

```diff
@@ -18,3 +18,8 @@
 
   <img style="max-width: 500px; width: 100%;" src="./interactive_tooltip.jpg" alt="Interactive tooltip example" />
 </ul>
+
+<h3>Themes</h3>
+<p>You can choose one of the themes to modify the look of the Tooltip.</p>
```

Comment:
Can you add a `Select` with themes in the docs examples of tooltips?

Reply:
sure